### ISSUE TITLE
Error about missing pipewire configuration on a system that does not have pipewire installed

### ISSUE THREAD

**Issue by pitsi** [mpv-player/mpv#14357]
### mpv Information

```bash
$ mpv --version
mpv 0.38.0 Copyright © 2000-2024 mpv/MPlayer/mplayer2 projects
libplacebo version: v6.338.2
FFmpeg version: 7.0.1
FFmpeg library versions:
   libavutil       59.8.100
   libavcodec      61.3.100
   libavformat     61.1.100
   libswscale      8.1.100
   libavfilter     10.1.100
   libswresample   5.1.100
```

### Other Information

- Linux version: 
```
PRETTY_NAME="Debian GNU/Linux trixie/sid"
NAME="Debian GNU/Linux"
VERSION_CODENAME=trixie
```
- Kernel Version: `Linux (removed) 6.7.12-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.7.12-1 (2024-04-24) x86_64 GNU/Linux`

- GPU Model: `01:00.0 VGA compatible controller [0300]: Advanced Micro Devices, Inc. [AMD/ATI] Caicos [Radeon HD 6450/7450/8450 / R5 230 OEM] [1002:6779]`
- Mesa/GPU Driver Version: `OpenGL version string: 4.5 (Compatibility Profile) Mesa 24.0.8-1`
- Window Manager and Version: `Openbox 3.6.1`
- Source mpv: `deb-multimedia.org`
- Introduced in version: `0.38`

### Reproduction Steps

On a system without pipewire (it may apply to pulseaudio as well), use mpv to play a file and notice the warnings seen below (they are colored in yellow and red respectively).

I used the --no-config option (which I did not know until a few minutes ago) and the warnings showed up the moment the video started playing. When using my config, the warnings show up when I press "i" (statistics overlay) and only once, no matter how many times I may press it during the playback.

Last but not least, in case it is not mpv's fault but something else, e.g. ffmpeg, I want to add that I am getting the same lines on the terminal when using kodi (21), but they appear there even when the app just launches and does nothing at all. Mpv is built with the libs of ffmpeg 7.0.1, but kodi is built with the libs of ffmpeg 6.1.1.

### Expected Behavior

```
$ mpv --no-config klama.mp4 
 (+) Video --vid=1 (*) (h264 800x600 29.970fps)
 (+) Audio --aid=1 (*) (aac 1ch 44100Hz)
File tags:
 Title: 1629350087352316
AO: [alsa] 48000Hz stereo 2ch float
VO: [gpu] 800x600 yuv420p
AV: 00:00:02 / 00:02:38 (2%) A-V:  0.000
Exiting... (Quit)
```

### Actual Behavior

```
$ mpv --no-config klama.mp4 
 (+) Video --vid=1 (*) (h264 800x600 29.970fps)
 (+) Audio --aid=1 (*) (aac 1ch 44100Hz)
File tags:
 Title: 1629350087352316
[W][02382.666467] pw.conf      | [          conf.c: 1031 try_load_conf()] can't load config client-rt.conf: No such file or directory
[E][02382.666681] pw.conf      | [          conf.c: 1060 pw_conf_load_conf_for_context()] can't load config client-rt.conf: No such file or directory
AO: [alsa] 48000Hz stereo 2ch float
VO: [gpu] 800x600 yuv420p
AV: 00:00:02 / 00:02:38 (2%) A-V:  0.000
Exiting... (Quit)
```

### Log File

[output.txt](https://github.com/user-attachments/files/15840716/output.txt)

### I carefully read all instruction and confirm that I did the following:

- [X] I tested with the latest mpv version to validate that the issue is not already fixed.
- [X] I provided all required information including system and mpv version.
- [X] I produced the log file with the exact same set of files, parameters, and conditions used in "Reproduction Steps", with the addition of `--log-file=output.txt`.
- [X] I produced the log file while the behaviors described in "Actual Behavior" were actively observed.
- [X] I attached the full, untruncated log file.
- [X] I attached the backtrace in the case of a crash.

**Comment by kasper93** [comment]
Duplicate of #11790

**Comment by Dudemanguy** [comment]
Duplicate #11790

**Comment by pitsi** [comment]
I did not think the issue would date back to mpv 0.35. On my end, it happened with the upgrade to 0.38, I did not have it before.

**Comment by kasper93** [comment]
Probably mpv was compiled without pipewire support before.

**Comment by pitsi** [comment]
> In the log you provided there is
> 
> ```
> [   0.000][v][cplayer] Setting option 'audio' = 'no' (flags = 12)
> ```
> 
> so no audio is loaded/used.

But... I do hear audio even with the --no-config option! Here is the proof that I do have alsa as my audio device in my mpv.conf
```
$ head -5 .config/mpv/mpv.conf 
vo=gpu,xv
#vo=xv
hwdec=vaapi
ao=alsa
sub-codepage=iso-8859-7
```

**Comment by pitsi** [comment]
Is there a command line parameter that can silence these errors only?

I am also interested in a command that will silence errors which come from ffmpeg, like this one
`[ffmpeg/demuxer] mpegts: Invalid timestamps stream=1, pts=7031650527, dts=7031654127, size=8198`
